### DISCUSSION TITLE
Reconstruction of a noisier sinogram gives the same image.

### DISCUSSION THREAD

**Discussion by yere-qubio** [UCL/STIR#1404] [Q&A] [answered]
Hi,

I'm doing PET simulation with SimSET. I've simulated two PET images, one with 300s of time acqusition and another one with 75s (25%). Altough the sinogram of the 75s image is noisier then the one of 300s, when I visualize both images with a viwer (MRICro) the images look the same. In addition, the SNR (Signar-to-Noise Ratio) is practically the same for both images.

Does someone faced something similar? Sorry, if the reason if something trivial for you.

You can find the sinograms, reconstructed images and OSMAPOSL parameter file [here](https://nubeusc-my.sharepoint.com/:f:/g/personal/jeremiah_poveda_rai_usc_es/Eijcl5jRPaJOn8INDkYCTiEB3AYY6fuJ1VSNe2fdU8Ocpw?e=lGa2Tq).

Thanks in advance.

**Comment by danieldeidda** [comment]
Hi,
I had a quick look at the images only and while they do look similar the numbers in it are quite different. Have a look at the line profiles. Not sure how noisy I would expect this to be because we don't know the number of simulated counts

**Reply by yere-qubio** [reply]
Here you have the detected photons (W) and the simulated decays (N_decays) in M (just look to 75s and 300s ignore the rest):

![image](https://github.com/UCL/STIR/assets/144900810/70f749a5-bf3b-4f78-a4d1-8a510301c2e6)

**Reply by danieldeidda** [reply]
I can see that you are using a big zoom in the parfile, maybe that is acting a bit like a filter. I would try with zoom =1.  Also 7M events it still quite a good number.

**Comment by KrisThielemans** [comment]
I haven't checked anything, but just a note that of course for initial iterations, noise isn't a big issue. You would see more differences at late iteration number.

**Comment by KrisThielemans** [comment, marked as answer]
Final note on this, depending on SimSET parameters, you might be using variance reduction, in which case SimSET will do its best to generate sinogram wiht much less noise than you expect from the number of counts..

**Reply by yere-qubio** [reply]
Totally, in addition, a big zoom was also contributing.
